parsing-stats/Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: all
+all:
+	./run-all
+
+.PHONY: test
+test:
+	$(MAKE) -C test
```

Comment:
Nitpick: maybe `test-lang` would be better?

Reply:
`test-lang` doesn't work because it's already file but yes, good idea. I'll change the folder name to `test-parsing`.

Reply:
oops sorry, it's `run-lang` that's already a file.